### PR TITLE
Scrapping escalation

### PR DESCRIPTION
**Pull request by Erick-INCS**
Hello!
Improved web scraping workflow to support an unlimited number of URLs and different sources :)

### CODE REVIEW

File: src/prospectScraper/Program.cs
Comment (zo0o0ot) [inline src/prospectScraper/Program.cs]: I don't think this works out as it did previously.

The order should be 

- 1
- 1b
- 2

etc.

It seems like the way it works currently is going from 1-6 then 1b, putting picks 17-32 at the end instead of between 16 and 33.

**Review by zo0o0ot — CHANGES_REQUESTED** [review]
Make sure that the changes here result in the same output as they did previously.  However, it does seem to clean up a fair bit of the code that I wasn't particularly proud of writing in my rush to get results from the scraper.

File: src/prospectScraper/mocks/2020-09-23-mock.csv
Comment (zo0o0ot) [inline src/prospectScraper/mocks/2020-09-23-mock.csv]: Here is the proof that the output has changed.

**Comment by zo0o0ot** [comment]
FYI, I merged a few things (including a relatively big refactor), so there's a chance that there's a fair bit of conflicts that I'd like to resolve before I merge this, if it's possible.

**Comment by Erick-INCS** [comment]
hello, already implement the same order

File: src/prospectScraper/Program.cs
Comment (zo0o0ot) [inline src/prospectScraper/Program.cs]: I noticed that MockDraftPicks is misspelled as MockDreaftPickLists - that should probably be updated, too.  Sorry for not noticing this previously.

**Review by Erick-INCS — COMMENTED** [review]
fix typo

**Pull request closed without merging**
